release-process.md

```diff
@@ -1,54 +1,131 @@
 # Release Process
 
-## Overview
+Our release process is how you deliver your work into the hands of end users. As part of [how we work](how-we-work.md), there are two key principles to give you full ownership of your work through the entire pipeline:
+
+
+- As an **individual engineer**, you're empowered and responsible to start release candidates on your own when you see fit.
+- As an **engineering team**, we're collectively empowered and responsible to reliably release our software to production.
+
+This is accomplished by decentralizing the process itself so that any engineer can bring their [production ready](production-ready.md) and [review approved](code-review.md) code to production.
+
+## Responsibilities
+
+### Individual Engineer
+
+As an engineer you have responsibilities to ensure the work you've done is reliably delivered to production:
+
 
-Our release process prepares an application for
-release to users. Releases are made in the context of a GitHub
-repository. This document describes the steps required.
+- Before your code is merged:
+  - Coordinate with DevOps on having configuration in place for your changes _before_ it gets deployed.
+- After your code is merged:
+  - Start a release candidate for your changes (and any other changes others may have committed) at some point within the next 12-24 hours of merging your changes.
+  - When the changes have been deployed on RC, give your changes a thorough round of testing.
+- After your code is released:
+  - When the release is deployed to production, monitor Sentry for possible errors that might indicate we need to revert.
+  - If you feel a release needs to be reverted, communicate this to the team and have an engineer who has permissions to do so take this action.
```

Comment:
Probably worth laying out the expected/accepted channel for this communication to take place so that there's no need to make that decision in the middle of firefighting.

Reply:
I think I'm going to open a separate issue for this, the changes here are adding up quite a bit.